hack/tests/scaffolding/scaffold-memcached.go

```diff
@@ -0,0 +1,298 @@
+// Copyright 2018 The Operator-SDK Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package main
+
+import (
+	"bytes"
+	"flag"
+	"fmt"
+	"io/ioutil"
+	"os"
+	"os/exec"
+	"path/filepath"
+	"regexp"
+	"strings"
+
+	"github.com/operator-framework/operator-sdk/internal/util/fileutil"
+
+	"github.com/rogpeppe/go-internal/modfile"
```

Comment:
While I find this project really interesting and useful! It seems a bit out of sync with the actual modfile internal package. Are we worried about this in any way?

Reply:
This was from the original `memcached_test.go` file and I just moved it over. There are some changes that I need to move into this PR though, and those remove our use of the `modfile` package, so this won't be problem once I resync this to master.